src/Fixer/ConstantNotation/NativeConstantInvocationFixer.php

```diff
@@ -225,6 +212,59 @@ protected function createConfigurationDefinition()
                 ->setAllowedValues([$constantChecker])
                 ->setDefault(['null', 'false', 'true'])
                 ->getOption(),
+            (new FixerOptionBuilder('scope', 'Only fix constant invocations that are made within a namespace or fix all.'))
+                ->setAllowedValues(['all', 'namespaced'])
+                ->setDefault('all')
+                ->getOption(),
         ]);
     }
+
+    private function fixConstantInvocations(Tokens $tokens, $start, $end)
```

Comment:
missing PHPDoc to typehint `$start` and `$end`

Reply:
the composer.json says that PHP-Cs-Fixer supports PHP 5.6+

Reply:
> PHPDoc

Reply:
hmm, sorry